controllers/datadogagent/utils.go

```diff
@@ -508,6 +508,21 @@ func getConfigInitContainers(spec *datadoghqv1alpha1.DatadogAgentSpec, volumeMou
 
 	copyCommands := []string{"cp -vnr /etc/datadog-agent /opt"}
 
+	if isComplianceEnabled(spec) && spec.Agent.Security.Compliance.ConfigDir != nil {
+		configVolumeMounts = append(
+			configVolumeMounts,
+			corev1.VolumeMount{
+				Name:      datadoghqv1alpha1.SecurityAgentComplianceCustomConfigDirVolumeName,
+				MountPath: "/etc/datadog-agent-compliance-benchmarks",
+			},
+			corev1.VolumeMount{
+				Name:      datadoghqv1alpha1.SecurityAgentComplianceConfigDirVolumeName,
+				MountPath: "/opt/datadog-agent/compliance.d",
+			},
+		)
+		copyCommands = append(copyCommands, "cp -v /etc/datadog-agent-compliance-benchmarks/* /opt/datadog-agent/compliance.d/")
+	}
```

Comment:
by doing files copy from the configmap in an init-container, it will require to delete/recreate the pod to get any updated made in the configmap. Is it what we want?

Reply:
You're right. This is not ideal but I don't think we modify this configmap very often (in fact, we haven't modified this configmap only once or twice in the last year)